src/main/java/com/programmerdan/minecraft/simpleadminhacks/configs/GameTuningConfig.java

```diff
@@ -108,6 +110,9 @@ protected void wireup(ConfigurationSection config) {
 
 		this.preventFallingThroughBedrock = config.getBoolean("preventFallingThroughBedrock", true);
 
+		this.badOmen = config.getBoolean("badOmen", false);
```

Comment:
This is defaulting to enabled, otherwise the PR is fine

Reply:
What do you mean by that ?

I'm testing it on my test server, leaving it undefined in the configuration file disables the effect.
Do you want it to be enabled by default ?